src/Interpreters/DatabaseCatalog.cpp

```diff
@@ -732,6 +784,10 @@ void DatabaseCatalog::dropTableDataTask()
             LOG_INFO(log, "Will try drop {}", table.table_id.getNameForLogs());
             tables_marked_dropped.erase(it);
         }
+        else
+        {
+            LOG_TRACE(log, "No tables to drop. Queue size: {}", tables_marked_dropped.size());
```

Comment:
The contradictory message, maybe something like: `"Not found any suitable tables to drop, still have {} tables in drop queue"`?